posthog/models/personal_api_key.py

```diff
@@ -39,7 +39,6 @@ class PersonalAPIKey(models.Model):
     id = models.CharField(primary_key=True, max_length=50, default=generate_random_token)
     user = models.ForeignKey("posthog.User", on_delete=models.CASCADE, related_name="personal_api_keys")
     label = models.CharField(max_length=40)
-    value = models.CharField(unique=True, max_length=50, editable=False, null=True, blank=True)
```

Comment:
My goal was to remove the reference to field now and then drop the column in a later PR. But the "Validate Django and CH migrations" GitHub Action complains if I don't drop this column in the same PR.

Is there any risk if we do these in the same PR? I imagine existing django web instances would error if the column was suddenly missing? Or do they only care if they actually try to use the column?